vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/pipeline_embedding_info.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_PIPELINE_PIPELINE_EMBEDDING_INFO_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_PIPELINE_PIPELINE_EMBEDDING_INFO_H_
+
+#include <map>
+#include <string>
+
+#include "bat/ads/internal/ml/ml_aliases.h"
+#include "bat/ads/internal/ml/data/vector_data.h"
+#include "bat/ads/internal/ml/model/linear/linear.h"
```

Comment:
not needed

Reply:
Removed reference to linear.h. I think the vector_data.h one should be there though?